src/client/sandbox/node/document/title-storage.ts

```diff
@@ -23,7 +23,9 @@ export default class DocumentTitleStorage extends EventEmitter {
         if (firstTitle)
             return firstTitle;
 
-        firstTitle = nativeMethods.createElement.call(this._document, 'title');
+        const createElement = (tagName => nativeMethods.createElement.call(this._document, tagName)) as Document['createElement'];
```

Comment:
I don't think that we need to create function for the type casting, in this case the `as HTMLTitleElement` is enough.